packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/FileDropZone/FileDropZone.tsx

```diff
@@ -43,7 +43,7 @@ export const BasicDemo: React.FunctionComponent = () => {
 
   return (
     <FileDropZone onFileDrop={handleFileDrop}>
-      <div style={{ border: '1px solid black', width: '100%', padding: '1rem' }}>
+      <div style={{ border: '1px solid black', height: '100vh', width: '100%', padding: '1rem' }}>
```

Comment:
Could you write a custom class here and move this style to the css?

Reply:
Just pushed something now!